scripts/video.js

```diff
@@ -154,6 +155,29 @@ H5P.Video = (function ($, ContentCopyrights, MediaCopyright, handlers) {
       return handlerName;
     };
 
+    /**
+    * @public
+    * Get current state for resetting it later.
+    *
+    * @returns {object} Current state.
+    */
+    self.getCurrentState = function () {
+      return {
+        startAt: self.getCurrentTime() || parameters.startAt
+      }
+    };
+
+    /**
+    * @public
+    * Reset task.
+    *
+    */
+    self.resetTask = function () {
+      parameters.startAt = 0;
```

Comment:
The `parameters.startAt` shouldn't be changed. This has been set by the author if they intend for the video to begin at a certain point.